### PR TITLE
Reformat nitrogen and phosphorus folder's docstring (1/2)

### PR DESCRIPTION
**Pull request by matthew7838**
### Summary
This PR will reformat the nitrogen folder and half of the phosphorus folder's docstring. 

### Notes
To keep the PR under 500 lines of changes, there will be a shorter follow-up PR regarding the second half of the phosphorus folder's docstring.

### CODE REVIEW

File: RUFAS/routines/field/soil/nitrogen_cycling/leaching_runoff_erosion.py [hunk 1]
```diff
@@ -237,21 +243,22 @@ def _determine_erosion_nitrogen_loss_content(
...
     ) -> float:
-        """This method determines nitrogen mass loss in erosion
+        """
+        This method determines nitrogen mass loss in erosion.
 
         Parameters
         ----------
         nitrogen_erosion_concentration: float
-            the soil nitrogen concentrations for the Fresh, Active, and Stable pools in soil (mg / kg)
+            the soil nitrogen concentrations for the Fresh, Active, and Stable pools in soil (mg / kg).
```
Comment (Isgotkowitz) [inline RUFAS/routines/field/soil/nitrogen_cycling/leaching_runoff_erosion.py]:
```suggestion
            The soil nitrogen concentrations for the Fresh, Active, and Stable pools in soil (mg / kg).
```

File: RUFAS/routines/field/soil/nitrogen_cycling/leaching_runoff_erosion.py [hunk 2]
```diff
@@ -237,21 +243,22 @@ def _determine_erosion_nitrogen_loss_content(
...
         daily_soil_lost: float
-            daily soil loss (Metric Tons / ha)
+            daily soil loss (Metric Tons / ha).
```
Comment (Isgotkowitz) [inline RUFAS/routines/field/soil/nitrogen_cycling/leaching_runoff_erosion.py]:
```suggestion
            Daily soil loss (Metric Tons / ha).
```

File: RUFAS/routines/field/soil/nitrogen_cycling/leaching_runoff_erosion.py [hunk 3]
```diff
@@ -262,17 +269,18 @@ def _determine_erosion_nitrogen_loss_content(
...
     def _determine_enrichment_ratio(daily_soil_lost: float) -> float:
-        """This method determines the enrichment ratio
+        """
+        This method determines the enrichment ratio.
 
         Parameters
         ----------
         daily_soil_lost: float
-            daily soil loss (Metric Tons/ha)
+            daily soil loss (Metric Tons/ha).
```
Comment (Isgotkowitz) [inline RUFAS/routines/field/soil/nitrogen_cycling/leaching_runoff_erosion.py]:
```suggestion
            Daily soil loss (Metric Tons/ha).
```

File: RUFAS/routines/field/soil/nitrogen_cycling/leaching_runoff_erosion.py [hunk 4]
```diff
@@ -262,17 +269,18 @@ def _determine_erosion_nitrogen_loss_content(
...
 
         Returns
         -------
         float
-            enrichment ratio (unitless)
+            enrichment ratio (unitless).
```
Comment (Isgotkowitz) [inline RUFAS/routines/field/soil/nitrogen_cycling/leaching_runoff_erosion.py]:
```suggestion
            Enrichment ratio (unitless).
```

**Review by Isgotkowitz — APPROVED** [review]
LGTM. Nice job Matthew.

**Review by allisterakun — APPROVED** [review]
Changes looks all good! Thank you for taking the time to go through those! Good job, Matthew!!!

**Pull request merged**